suprb/optimizer/rule/mutation.py

```diff
@@ -69,47 +87,108 @@ class Normal(RuleMutation):
     def ordered_bound(self, rule: Rule, random_state: RandomState):
         # code inspection gives you a warning here but it is ineffectual
         rule.match.bounds += random_state.normal(scale=self.sigma,
-                                        size=rule.match.bounds.shape)
+                                                 size=rule.match.bounds.shape)
         rule.match.bounds = np.sort(rule.match.bounds, axis=1)
 
+    def unordered_bound(self, rule: Rule, random_state: RandomState):
+        rule.match.bounds += random_state.normal(scale=self.sigma,
+                                                 size=rule.match.bounds.shape)
```

Comment:
As `ordered_bound` is just the sorted array of `unordered_bound` you could use `unordered_bound` in `ordered_bound` instead of  duplicating this line:

```
rule.match.bounds += random_state.normal(scale=self.sigma,
                                                 size=rule.match.bounds.shape)
```